lib/ruby_lsp/check_docs.rb

```diff
@@ -34,6 +35,13 @@ def define_task
       task(@name) { run_task }
     end
 
+    sig { params(request_path: String).returns(T::Boolean) }
+    def gif_exists?(request_path)
+      request_gif = request_path.gsub(".rb", ".gif").split("/").pop
```

Comment:
```suggestion
      request_gif = request_path.gsub(".rb", ".gif").split("/").last
```
(since there's no need to mutate `request_path`).